concourse/scripts/parse_buildpacks_and_run_uploader.rb

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+require 'yaml'
+
+unless ARGV[0]
+  puts "USAGE: #{$PROGRAM_NAME} /path/to/buildpacks.yml"
+  exit 1
+end
+
+yaml = YAML.safe_load(File.read(ARGV[0].to_s))
+yaml['buildpacks'].each do |bp|
+  system("./paas-cf/concourse/scripts/upload-buildpack.sh", \
```

Comment:
Do we need the backslashes on the end of these lines?

Reply:
Good spot, we don't - this is ruby, not bash...

Reply:
This was originally all inlined. On it.